testsuite/tests/tool-ocamlobjinfo/Makefile

```diff
@@ -0,0 +1,47 @@
+BASEDIR=../..
+
+LD_PATH=
+
+.PHONY: default
+default:
+	@if ! $(SUPPORTS_SHARED_LIBRARIES) || $(BYTECODE_ONLY) ; then \
+	  echo 'skipped (.cmxs not built)'; \
+	elif ! grep -q HAS_LIBBFD $(TOPDIR)/config/s.h ; then \
+	  echo 'skipped (BFD library not available)'; \
+	else \
+	  case " $(OTHERLIBRARIES) " in \
+	    *' unix '*) \
+	       $(SET_LD_PATH) OCAMLLIB=$(TOPDIR)/tools $(MAKE) run;; \
+	    *' win32unix '*) \
+	       $(SET_LD_PATH) OCAMLLIB=$(TOPDIR)/tools $(MAKE) run;; \
+	    *) echo 'skipped (unix.cmxs not available)';; \
+	  esac \
+	fi
+
+.PHONY: run
+run:
+	@printf " ... testing 'ocamlobjinfo'"
+	@rm -f $(MAIN_MODULE).result
+	@$(OCAMLRUN) `$(CYGPATH) $(TOPDIR)/tools/ocamlobjinfo$(EXE)` \
+	             $(OTOPDIR)/otherlibs/$(UNIXLIBVAR)unix/unix.cmxs \
+	             > test.raw.result 2>&1 \
+	 && sed -e 's/[0-9a-z]\{32\}/<MD5>/' \
```

Comment:
Should you not test for spaces, or at least non-word characters, around the pattern? It work in practice because we don't have alphanumeric words that long (especially without uppercase letters) but it's a bit weird to rely on this.